meinberlin/apps/notifications/signals.py

```diff
@@ -18,13 +19,9 @@
 def send_notifications(instance, created, **kwargs):
     action = instance
```

Comment:
as this is now running some rather heavy database queries, can we move the content of this signal into a celery task, and the signal here just triggers that task?